test/stdlib/StringBridge.swift

```diff
@@ -75,6 +75,47 @@ StringBridgeTests.test("Tagged NSString") {
 #endif // not 32bit
 }
 
+#if _runtime(_ObjC)
```

Comment:
This whole file is `REQUIRES: objc_interop`, at least for now.

Reply:
ah, right. Thank you